src/web/app/components/question-submission-form/question-submission-form.component.html

```diff
@@ -161,6 +161,11 @@
     </div>
 
     <div class="col-12 constraint-margins">
+      <tm-contribution-question-constraint *ngIf="model.questionType === FeedbackQuestionType.CONTRIB"
+                                                  [recipientSubmissionForms]="model.recipientSubmissionForms"
+                                                  [questionDetails]="model.questionDetails"
+                                                  (isValidEvent)="updateValidity($event)"
```

Comment:
nit: indentation for these should be in line with `*ngIf` (following other examples in this file)